web/pdf_sidebar.js

```diff
@@ -19,10 +19,12 @@ import { RenderingStates } from './pdf_rendering_queue';
 const UI_NOTIFICATION_CLASS = 'pdfSidebarNotification';
 
 const SidebarView = {
+  UNKNOWN: -1,
   NONE: 0,
-  THUMBS: 1,
+  THUMBS: 1, // Default value.
   OUTLINE: 2,
   ATTACHMENTS: 3,
+  LAYERS: 4,
```

Comment:
Since I needed to touch this enum anyway, I figured it couldn't hurt to just add this for a possible future "layers" feature (or Optional Content Groups[1] as the proper name is).

This is kind of similar to one of the cursor tools, i.e. `ZOOM`, which also isn't implemented yet: https://github.com/mozilla/pdf.js/blob/9f9d87c3ceae59e61b0095a6cb739908138c0799/web/pdf_cursor_tools.js#L18-L22

---
[1] Some time ago I glanced at the relevant sections of the PDF specification, wondering how hard this would be to implement, but stopped reading when coming across text suggesting that some operators should be executed even while that part of the `Content` stream was being ignored :-(